torch/utils/data/dataset.py

```diff
@@ -17,19 +14,12 @@
 # No 'default_generator' in torch/__init__.pyi
 from torch import default_generator, randperm
 from torch._utils import _accumulate
-from torch.utils.data._typing import _DataPipeMeta
-from torch.utils.data._utils.serialization import serialize_fn, deserialize_fn, SerializationType
 
 from ... import Generator, Tensor
 
 T_co = TypeVar('T_co', covariant=True)
 T = TypeVar('T')
 
-UNTRACABLE_DATAFRAME_PIPES = ['batch',  # As it returns DataChunks
-                              'groupby',   # As it returns DataChunks
-                              '_dataframes_as_tuples',  # As it unpacks DF
-                              'trace_as_dataframe',  # As it used to mark DF for tracing
-                              ]
 
 class DataChunk(list, Generic[T]):
     def __init__(self, items):
```

Comment:
Is that possible to move `DataChunk` out of `dataset.py` as well?

Reply:
Yep, I just did and updated this PR.